io/module_wrt_grid_comp.F90

```diff
@@ -2448,7 +2447,7 @@ subroutine wrt_run(wrt_comp, imp_state_write, exp_state_write,clock,rc)
 
           if (out_phase == 2 .and. restart_written .and. mype == lead_write_task) then
             !**  write coupler.res log file
-            open(newunit=nolog, file='RESTART/'//trim(time_restart)//'.coupler.res', status='new')
```

Comment:
So this change allows the restart files to be overwritten?

Reply:
This is just to make rerunning the model easier in the same run directory without needing to remove the 'coupler.res' file from the previous run. Other restart files can already be overwritten.